app/src/main/java/com/example/playlistmaker/feature_medialibrary/presentation/MediaLibraryActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.playlistmaker.feature_medialibrary.presentation
+
+import android.os.Bundle
+import androidx.activity.enableEdgeToEdge
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.updatePadding
+import androidx.viewpager2.widget.ViewPager2
+import com.example.playlistmaker.databinding.ActivityMediaLibraryBinding
+import com.google.android.material.tabs.TabLayoutMediator
+import com.example.playlistmaker.R
+
+class MediaLibraryActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMediaLibraryBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        enableEdgeToEdge()
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityMediaLibraryBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setupEdgeToEdge()
+        setupViewPager()
+        setupClickListeners()
+    }
+
+    private fun setupEdgeToEdge() {
+        ViewCompat.setOnApplyWindowInsetsListener(binding.root) { view, insets ->
+            val statusBarInsets = insets.getInsets(WindowInsetsCompat.Type.statusBars())
+            val navigationBarInsets = insets.getInsets(WindowInsetsCompat.Type.navigationBars())
+
+            view.updatePadding(
+                top = statusBarInsets.top,
+                bottom = navigationBarInsets.bottom
+            )
+
+            insets
+        }
+    }
+
+    private fun setupViewPager() {
+        val adapter = MediaLibraryPagerAdapter(this)
+        binding.viewPager.adapter = adapter
+
+        TabLayoutMediator(binding.tabLayout, binding.viewPager) { tab, position ->
+            tab.text = when (position) {
+                0 -> getString(R.string.favorite_tracks)
+                1 -> getString(R.string.playlists)
+                else -> ""
```

Comment:
Здесь можно упростить:

```suggestion
                else -> getString(R.string.playlists)
```